create_bash_script.py

```diff
@@ -1,54 +1,74 @@
+import os
 import sys
 
-with open('docker/Dockerfile', 'r') as f:
+
+with open("docker/Dockerfile", "r") as f:
     dockerfile_lines = f.readlines()
 
-bash_instructions = ['#!/bin/bash']
-docker_command = ''
+claasp_directory = os.path.abspath(".")
+bash_instructions = ["#!/bin/bash"]
 environment_variables = []
+docker_command = ""
+
 for line in dockerfile_lines:
     line = line.strip()
 
-    if line.startswith("FROM claasp-base AS claasp-lib"):
-        break
+    if line.startswith("FROM"):
```

Comment:
This _if line.startswith("FROM"):_ is not necessary as it will enter in the last _case_ of the _match_ sentence. The only needed part is to specify to stop the reading once the specific image _claasp-lib_ is triggered.